### PR TITLE
[CORE-502] DealSources.list

### PR DESCRIPTION
**Pull request by JordyDeGroeve**

### CODE REVIEW

**Review by dietrichm — DISMISSED** [review]

**Review by WouterSioen — DISMISSED** [review]

**Review by mansilaxel — DISMISSED** [review]

File: apiary.apib [hunk 1]
```diff
@@ -1179,6 +1179,26 @@ Get a list of all phases a deal can go through, sorted by their order in the flo
...
+### dealSources.list [GET /dealSources.list]
+
+Get a list of all deal sources, sorted alphabetically (on description)
+
++ Request (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + filter (object, optional)
+            + ids: `4bd112da-8a39-47b9-b1e2-8ce339c241a3`,`94bd8c65-72ce-4a43-9cdf-9b75f1966010` (array[string], optional)
+        + page (Page, optional)
```
Comment (jenssegers) [inline apiary.apib]: Can we add the default sorting?

**Review by jenssegers — DISMISSED** [review]
Can we add default sorting to the request attributes?

**Review by WouterSioen — DISMISSED** [review]

**Review by dietrichm — DISMISSED** [review]

**Review by mansilaxel — DISMISSED** [review]

**Review by WouterSioen — DISMISSED** [review]
🙌

**Review by dietrichm — DISMISSED** [review]

**Review by mrtus — DISMISSED** [review]

**Review by dietrichm — APPROVED** [review]

**Review by mansilaxel — APPROVED** [review]

**Review by WouterSioen — APPROVED** [review]

**Comment by marijnvandevoorde** [comment]
Merge away @JordyDeGroeve

**Pull request merged**